tests/integration/advanced/test_timeseries.py

```diff
@@ -209,7 +198,7 @@ def test_3_time_series_sktime_mixer(self):
         code = code_from_json_ai(json_ai)
         predictor = predictor_from_code(code)
 
-        predictor.learn(train)
+        train_and_check_time_aim(predictor, train)
```

Comment:
The one big worry I have is that I don't see (yet) a way to make `sktime` compliant with time aim... I'll give it some more thought, but maybe for now a disclaimer at the start of its `fit()` method could be good for the user to notice.